bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -48,7 +48,8 @@ def execute(self):
         self.__install_build_dependencies()
         for config in self.test_config.integ_test["test-configs"]:
             security = self.__is_security_enabled(config)
-            self.__setup_cluster_and_execute_test_config(security)
+            status = self.__setup_cluster_and_execute_test_config(security)
+            return status, config
```

Comment:
This is incorrect. It will return before the for loop is able to execute all configs.

Reply:
Addressed in https://github.com/opensearch-project/opensearch-build/pull/630